src/jit/morph.cpp

```diff
@@ -6163,6 +6163,14 @@ GenTreePtr Compiler::fgMorphField(GenTreePtr tree, MorphAddrContext* mac)
             return newTree;
         }
     }
+    else if ((objRef != nullptr) && (objRef->OperGet() == GT_ADDR) && varTypeIsSIMD(objRef->gtGetOp1()))
```

Comment:
I hit cases before where we were getting into MACK_Ind contexts for Vector3 field access unexpectedly, and I fixed those reasons, so I'm worried this is treating the symptom, not the cause. However, if you believe this is a good, conservative fix, then I'm ok with it.